### DISCUSSION TITLE
What's the "k" in SAGEConv

### DISCUSSION THREAD

**Discussion by Turninger** [pyg-team/pytorch_geometric#2649] [Q&A]
According to the Paper of GraphSAGE 
![image](https://user-images.githubusercontent.com/52813653/119959732-dd33d400-bfd6-11eb-8403-7c98517c3598.png)
there will be **K** time for aggregator, but I can't find the **K** in pytorch_geometic SAGEConv.
Anybody knows what the **K** is ? thanks alot!

**Comment by rusty1s** [comment]
The `k` in this algorithm denotes the current layer, e.g., `h_v^{k-1}` denotes the representation of node v after the `k-1` layer.
